CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased] - ReleaseDate
 ### Added
+- Added linux TIMESTAMPNS support for x86/x64 gnu
```

Comment:
Why only x86_64 and x86?  It looks like that symbol is defined for many other arches, too.

Reply:
Even now arch like `Linux i686` is still failing. I am wondering why.

Reply:
It seems that even i686 linux does not have that symbol. Neither did mips, etc...

I have restricted to x86_64, and now the tests pass.